packages/input-option/src/InputOption/InputOption.types.ts

```diff
@@ -9,12 +11,7 @@ import { DarkModeProps } from '@leafygreen-ui/lib';
  *
  * `active`: The element is selected, or otherwise active (including `:active`)
  */
-export interface BaseInputOptionProps {
-  /**
-   * Content to appear inside of option
-   */
-  children?: React.ReactNode;
-
+export interface BaseInputOptionProps extends PropsWithChildren<{}> {
```

Comment:
i think the proper way to write this would be

`interface BaseInputOptionProps {}`

`type InputOptionProps = PropsWithChildren<BaseInputOptionProps>`